### DISCUSSION TITLE
Support for Redis 7.1

### DISCUSSION THREAD

**Discussion by AlexandruHodorogea** [alisaifee/coredis#223] [Q&A]
Hi!

Does anyone have experience using the coredis library with Redis 7.1? 

I see in the documentation that the library has been tested only for 7.0 and 7.2, is there any particular reason 7.1 was not included?

Our Redis provider has vector search support only for 7.1 and the 7.2 version has not been released yet. I was wondering if we could use the client with the 7.1 setup.

Any input would help

Thanks!

**Comment by alisaifee** [comment]
Apologies for not getting to this. 7.1 is definitely supported, it was just not part of the CI matrix. It is now.
